packages/core/src/Parser.ts

```diff
@@ -11,6 +11,9 @@ import { FragmentNode } from './VNodes/FragmentNode';
 import { FormatType, CharNode } from './VNodes/CharNode';
 import { ListNode } from './VNodes/ListNode';
 
+const listTags = ['UL', 'OL'];
+const listContextTags = listTags.concat('LI');
```

Comment:
I don't understand. How does that help with the agnostic parser? This should still be read from the List plugin wouldn't it?